portfolio.html

```diff
@@ -70,9 +70,10 @@ <h3>
         >
       </h3>
     </footer>
-    <script src="javascript.js"></script>
+    <script src="index.js"></script>
     <script>
-      fetchLocalStorageData();
+      renderExperienceList();
+      fetchExperienceList();
```

Comment:
do we need to call these fucntions here explicitly like this? cant we call them when the page is loaded i.e by adding an eventListener for DOMContentLoaded ? like: 
`document.addEventListener("DOMContentLoaded", function () {
 yourFunction();
 }
 `

Reply:
resolved!